docs/includes/pairlists.md

```diff
@@ -74,6 +75,15 @@ be caught out buying before the pair has finished dropping in price.
 
 This filter allows freqtrade to ignore pairs until they have been listed for at least `min_days_listed` days.
 
+#### PerformanceFilter
+
+Sorts pairs by performance, as follows:
```

Comment:
maybe 

```suggestion
Sorts pairs by past trade performance, as follows:
```

to make it clear that it's from the trades performance (als simple historic returns can be seen as one form of performance ...)

Apart from this i think this is ok now (unless you think there's still something missing)

Reply:
It's good to go